### PR TITLE
[GNA] Added support configuration of nthreads and quantization bits for python benchmark

### PR DESCRIPTION
**Pull request by admitriev-gna**

### CODE REVIEW

File: tools/benchmark/main.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from openvino.tools.benchmark.benchmark import Benchmark
 from openvino.tools.benchmark.parameters import parse_args
 from openvino.tools.benchmark.utils.constants import MULTI_DEVICE_NAME, HETERO_DEVICE_NAME, CPU_DEVICE_NAME, \
-    GPU_DEVICE_NAME, MYRIAD_DEVICE_NAME, BIN_EXTENSION, BLOB_EXTENSION
+    GPU_DEVICE_NAME, MYRIAD_DEVICE_NAME, BIN_EXTENSION, BLOB_EXTENSION, GNA_DEVICE_NAME
```
Comment (dorloff) [inline tools/benchmark/main.py:8]: Why not put GNA_DEVICE_NAME after other DEVICE_NAMEs?

File: tools/benchmark/utils/constants.py [hunk 2]
```diff
@@ -41,6 +42,7 @@
     MYRIAD_DEVICE_NAME: 60,
     HDDL_DEVICE_NAME: 60,
     FPGA_DEVICE_NAME: 120,
+    GNA_DEVICE_NAME: 120,
```
Comment (dorloff) [inline tools/benchmark/utils/constants.py:45]: Why 120, not 60?

File: tools/benchmark/main.py [hunk 3]
```diff
@@ -154,6 +154,14 @@ def set_throughput_streams():
                     config[device]['CLDNN_PLUGIN_THROTTLE'] = '1'
             elif device == MYRIAD_DEVICE_NAME:
                 config[device]['LOG_LEVEL'] = 'LOG_INFO'
+            elif device == GNA_DEVICE_NAME:
+                if is_flag_set_in_command_line('qb'):
+                    if args.qb == 8:
+                        config[device]['GNA_PRECISION'] = 'I8'
+                    else:
+                        config[device]['GNA_PRECISION'] = 'I16'
+                if args.number_threads and is_flag_set_in_command_line("nthreads"):
```
Comment (dorloff) [inline tools/benchmark/main.py:163]: Should be reflected in description of command line parameters

File: tools/benchmark/parameters.py [hunk 4]
```diff
@@ -107,6 +107,8 @@ def parse_args():
     args.add_argument('-load_config', type=str, required=False, default='',
                       help="Optional. Path to JSON file to load custom IE parameters."
                            " Please note, command line parameters have higher priority then parameters from configuration file.")
+    args.add_argument('-qb', type=int, required=False, default=False,
```
Comment (dorloff) [inline tools/benchmark/parameters.py:110]: Where is the input value validated?

File: tools/benchmark/parameters.py [hunk 5]
```diff
@@ -107,6 +109,8 @@ def parse_args():
     args.add_argument('-load_config', type=str, required=False, default='',
                       help="Optional. Path to JSON file to load custom IE parameters."
                            " Please note, command line parameters have higher priority then parameters from configuration file.")
+    args.add_argument('-qb', '--quantization_bits', type=int, required=False, default=None,
+                      help="Optional. Weight bits for quantization:  8 (I8) or 16 (I16) ")
```
Comment (nkogteva) [inline tools/benchmark/parameters.py:113]: I propose to add `choices=[8, 16]`

File: tools/benchmark/parameters.py [hunk 6]
```diff
@@ -20,6 +20,8 @@ def validate_args(args):
         raise Exception("Number of inference requests should be positive (invalid -nireq option value)")
     if not (fnmatch(args.path_to_model, XML_EXTENSION_PATTERN) or fnmatch(args.path_to_model, BLOB_EXTENSION_PATTERN)):
         raise Exception('Path {} is not xml or blob file.')
+    if args.quantization_bits and args.quantization_bits != 16 and args.quantization_bits != 8:
```
Comment (nkogteva) [inline tools/benchmark/parameters.py:23]: With `choices` these lines are not needed

**Comment by dorloff** [comment]
I think it needs to be rebased.

**Pull request merged**